packages/strapi-plugin-i18n/services/locales.js

```diff
@@ -10,10 +12,13 @@ const create = locale => strapi.query('locale', 'i18n').create(locale);
 
 const update = (params, updates) => strapi.query('locale', 'i18n').update(params, updates);
 
+const setDefaultLocale = ({ code }) => getCoreStore().set({ key: 'default_locale', value: code });
```

Comment:
you can add a getDefaultLocale too as we are going to need to get it pretty often :)